src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  font-family: Roboto, Helvetica, sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: rgb(255,255,255);
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+    display: block;
+  }
+}
+
+.nav-list {
```

Comment:
`.nav-list` is child of `.header` - please use SCSS nesting accordingly

Reply:
done in fixed solution